lms/djangoapps/grades/models.py

```diff
@@ -138,14 +141,15 @@ class VisibleBlocks(models.Model):
     blocks_json = models.TextField()
     hashed = models.CharField(max_length=100, unique=True)
     course_id = CourseKeyField(blank=False, max_length=255, db_index=True)
+    version = models.IntegerField()
```

Comment:
Should there be a new index that includes version, or have a single index and we always query with version?